src/plugins/intel_cpu/src/nodes/mkldnn_rnn.cpp

```diff
@@ -424,7 +477,7 @@ void MKLDNNRNN::fillSequenceDesc() {
     outCandidate.reserve(3);
 
     if (nativeOrder) {
-        outCandidate.emplace_back(std::make_shared<DnnlBlockedMemoryDesc>(outDataDescs[RNNInOutKind::Layer]));
+        outCandidate.emplace_back(std::make_shared<DnnlBlockedMemoryDesc>(*outDataDescs[RNNInOutKind::Layer]));
```

Comment:
Can we just copy the pointer here?

Reply:
Const and not const types conflict.

Reply:
Though it is not very important problem, but const cast can be used for that purpose. Or, we can store pointers on non costant type in outDataDescs and inDataDescs

Reply:
Reverted to non const.